package/thingino-webui/files/var/www/x/j/reset-ntp.cgi

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-if cp /rom/etc/ntp.conf /etc/ntp.conf; then
+if cp /rom/etc/ntp.conf /tmp/ntp.conf; then
```

Comment:
Same question here.  /etc/ntp.conf will be a symlink to /tmp/ntp.conf.  What is at /rom/etc/ntp.conf?